src/Repository/Repository.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Repository;
+
+/**
+ * @author Ener-Getick <egetick@gmail.com>
+ */
+class Repository
+{
+    /**
+     * @var string
+     */
+    private $vendor;
+
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var string|null
+     */
+    private $secret;
```

Comment:
None for now.
I wanted to securize the api (anyone who knows its url can alter sf pull requests) but i decided to do it later.
We can remove this for now.